toolkits/springer/packages/springer-header/README.md

```diff
@@ -16,12 +16,12 @@ Publisher level header for Springer products.
             <!-- #not mandatory -->
             <a href="#"
                class="c-header__link c-header__link--search">
-                <span class="u-flex u-flex-align-center">
-                    Search
-                    <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true" focusable="false">
-                        <use xlink:href="#icon-search"></use>
-                    </svg>
-                </span>
+				<span class="u-flex u-flex-align-center">
+					Search
+					<svg class="u-icon u-ml-4" width="22" height="22" aria-hidden="true" focusable="false">
```

Comment:
Should we be including the width and height attributes on this <svg> in light of the recent discussions about `1em` width and height being applied to `u-icon`?

Reply:
@benjclark That's a very good question! I'm trying my best to keep PRs focused, but I think this can be updated, yeah.